### ISSUE TITLE
eas credentials logging into apple leads to Error: unable to get local issuer certificate

### ISSUE THREAD

**Issue by dodyeid** [expo/eas-cli#2472]
### Summary

When I run eas credentials, refuse to log in, then for iOS I choose Generate a new Apple Distribution Certificate?, then I successfully log in to apple account, but I immediately get: "Authentication with Apple Developer Portal failed!
Error: unable to get local issuer certificate"

I had an expired certificate that I've removed. When I run "Distribution Certificate: Delete one from your account" I get
"There are no Distribution Certificates available in your EAS account."

Thanks.

### Managed or bare?

Managed

### Environment

expo-env-info 1.2.0 environment info:
    System:
      OS: Windows 11 10.0.22631
    Binaries:
      Node: 20.15.1 - C:\Program Files\nodejs\node.EXE
      npm: 10.8.2 - ~\triviaLink\node_modules\.bin\npm.CMD
    IDEs:
      Android Studio: AI-222.4459.24.2221.9862592
    npmPackages:
      @expo/metro-config: ^0.18.4 => 0.18.8
      expo: ^51.0.20 => 51.0.20
      react: 18.2.0 => 18.2.0
      react-dom: 18.2.0 => 18.2.0
      react-native: 0.74.3 => 0.74.3
      react-native-web: ~0.19.10 => 0.19.12
    Expo Workflow: managed
    
    
    Expo doctor output:
    ✔ Check Expo config for common issues
✔ Check package.json for common issues
✔ Check native tooling versions
✖ Check dependencies for packages that should not be installed directly
✔ Check for common project setup issues
✔ Check npm/ yarn versions
✔ Check for issues with metro config
✖ Check Expo config (app.json/ app.config.js) schema
Unexpected error while running 'Check Expo config (app.json/ app.config.js) schema' check:
TypeError: fetch failed
✔ Check for legacy global CLI installed locally
✔ Check that native modules do not use incompatible support packages
✖ Check that native modules use compatible support package versions for installed Expo SDK
✖ Check that packages match versions required by installed Expo SDK

Detailed check results:

Expected package @expo/config-plugins@~8.0.0
Found invalid:
  @expo/config-plugins@5.0.4
  (for more info, run: npm why @expo/config-plugins)
Expected package @expo/prebuild-config@~7.0.0
Found invalid:
  @expo/prebuild-config@5.0.7
  (for more info, run: npm why @expo/prebuild-config)
Advice: Upgrade dependencies that are using the invalid package versions.

The package "npm" should not be installed directly in your project. It may conflict with the globally-installed version.
The package "expo-app-loading" has been removed as of SDK 49. You should use expo-splash-screen instead. Learn more: https://docs.expo.dev/versions/latest/sdk/splash-screen/

The following packages should be updated for best compatibility with the installed expo version:
  expo@51.0.20 - expected version: ~51.0.22
  expo-font@12.0.8 - expected version: ~12.0.9
  expo-notifications@0.28.9 - expected version: ~0.28.12
  expo-updates@0.25.19 - expected version: ~0.25.21
  react-native-safe-area-context@4.10.1 - expected version: 4.10.5
Your project may not work correctly until you install the expected versions of the packages.
Found outdated dependencies

### Reproducible demo or steps to reproduce from a blank project

Run eas credentials, try to generate a new apple distribution certificate.

**Comment by szdziedzic** [comment]
Hi @dodyeid,

I believe that `unable to get local issuer certificate` is an SSL error and not an error from Apple's API, so I think that it has nothing to do with distribution certificates 🤔. 

Can you run the command with `EXPO_DEBUG=1` and `DEBUG=*` env variables set to get more logs?

**Comment by dodyeid** [comment]
Hi and thanks for your help @szdziedzic ! I ran `set EXPO_DEBUG=1` , followed by  `set DEBUG=*`, and then ran `eas credentials` again followed by logging in only to get the same error message (nothing new or added popped up). I am a "noob" so perhaps I am not properly setting the env variables.

One thing to note that might help you/us - when I copied my project repo on another computer, installed node, expo, and eas, and then ran `eas credentials` there, it worked perfectly fine! So I think something is occurring with my particular system / system metadata.

Thanks again for your help.
